extensions/vscode/package.json

```diff
@@ -441,25 +449,30 @@
           "name": "Log",
           "contextualTitle": "Log",
           "icon": "$(output)",
-          "when": "workbenchState == folder && posit.publish.state == 'initialized'"
+          "when": "!isWorkspaceTrusted"
```

Comment:
This looks like a mistake. It looks like this will need to change due to the work you just merged to `main` though.

Reply:
Great catch! Thanks!